### PR TITLE
Allows get sheet or sheet index by sheet name

### PR DESCRIPTION
**Pull request by fabiofsilva**
Hi, 
This pull request is referent to #66, that was closed due lack of feedback. I've made the changes that you requested. Sorry, but I was working in another project.

### CODE REVIEW

File: tests/test_by_name_functions.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        'Get sheet by index'
+        ws = self.wb.get_sheet('Plan2')
+        self.assertEqual('Plan2', ws.name)
+    
+    def test_invalid_sheet_parameter(self):
+        'Raises exception when sheet is not string or integer'
+        try:
+            self.wb.get_sheet(1.1)
+        except Exception as e:
+            self.assertTrue('sheet must be integer or string', e)
```
Comment (cjw296) [inline tests/test_by_name_functions.py]: This isn't asserting what you think it's asserting...
(hint: put `e=None` on the line above...)

Comment (fabiofsilva) [inline tests/test_by_name_functions.py]: Sorry, I did not understand what you mean by putting e=None.

Comment (cjw296) [inline tests/test_by_name_functions.py]: Then please go and read the documentation for assertTrue so you understand why it's the wrogn assertion method to use here. Hint: your current assertion will *always* pass. When you write tests, make sure they fail *before* fixing the code...

Comment (fabiofsilva) [inline tests/test_by_name_functions.py]: You where rigth, my mistake :). I was trying to test the exception message. I'll test only if the exception is raised then.

**Comment by fabiofsilva** [comment]
Fixed on last commit.

**Pull request merged**